models/IM-resolved.yaml

```diff
@@ -0,0 +1,54 @@
+description: "Two-deme isolation-with-migration model."
+time_units: generations
+demes:
+- name: A
+  description: "The ancestral deme"
+  start_time: .inf
```

Comment:
This should be "Infinity" I think?

Reply:
Only for the JSON output. YAML `.inf` is well defined.